packages/nitro-protocol/scripts/postdeploy

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cat addresses.json | jq 'walk(if (type == "object" and .abi) then del(.abi) else . end )' > addresses.json.tmp
```

Comment:
What happens if jq is not installed locally?

Reply:
I hit this problem and decided to just rewrite what we need in javascript: 
https://github.com/statechannels/statechannels/pull/3072: https://github.com/statechannels/statechannels/pull/3072/commits/444c6130ee405a6bc1b647168e91c79e87a46223

Reply:
@snario is `addresses.json` included in the npm package?